lib/Logger.js

```diff
@@ -77,13 +86,15 @@ class Logger {
         const rLog = new RequestLogger(Config.logger,
                                        Config.level, Config.dump, Config.end,
                                        unserializeUids(serializedUids));
-        rLog.addDefaultFields({ name: this.name });
+        rLog.setParent(this);
         return rLog;
     }
 
     _doLog(levelName, msg, data) {
         const sLogger = Config.logger;
-        const finalData = { name: this.name, time: Date.now() };
+        const finalData = objectCopy({},
```

Comment:
That would mean updating the default value of one logger element, thus updating all of its children logger default fields as a side-effect for EACH logging request. This would be so much more consuming than only doing an object copy at the time of the request. Imagine updating all of the precomputedFields of all RequestLoggers whenever you log something with the root module...
Also, that's how the RequestLogger has always been working, no reason to change ways when applying the scheme to the ModuleLogger